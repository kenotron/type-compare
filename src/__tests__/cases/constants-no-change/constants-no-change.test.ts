import { extractTestCaseName, testHarness } from "../../test-harness";
testHarness(extractTestCaseName(__filename));
