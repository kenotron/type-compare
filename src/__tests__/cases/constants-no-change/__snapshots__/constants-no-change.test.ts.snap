// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case "constants-no-change" 1`] = `
{
  "messages": [],
  "minChangeType": "none",
}
`;
